src/opm/output/eclipse/WriteRFT.cpp

```diff
@@ -347,6 +348,7 @@ namespace {
     }
 
     void RFTRecord::addConnection(const ::Opm::UnitSystem&       usys,
+                                  const double cell_depth, 
                                   const ::Opm::Connection&       conn,
```

Comment:
With this change, the `conn` parameter becomes unused.  I think we should just remove it now.

Reply:
Good point, removed now.